jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -194,9 +194,7 @@
             colormap: xterm
     builders:
         - shell: 'JENKINS_BUILD_STARTED=true bash <(curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-to-gcs.sh")'
-        - shell: |
-            export KUBE_VERIFY_GIT_BRANCH="${ghprbTargetBranch}"
-            ./hack/jenkins/gotest-dockerized.sh
+        - shell: '{cmd}'
     publishers:
         - xunit:
```

Comment:
It is until https://github.com/kubernetes/kubernetes/pull/27289 is merged, so we won't know.